test/agentchat/contrib/test_postgresql_agent.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python3 -m pytest
+
+import os
+import sys
+
+import pytest
+
+from autogen import config_list_from_json
+
+sys.path.append(os.path.join(os.path.dirname(__file__), "../.."))
+from conftest import skip_openai  # noqa: E402
+
+sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
+from test_assistant_agent import KEY_LOC, OAI_CONFIG_LIST  # noqa: E402
+
+from autogen import AssistantAgent
+from autogen.coding.markdown_code_extractor import MarkdownCodeExtractor
+
+try:
+    import psycopg
+
+    from autogen.agentchat.contrib.postgresql_agent import PostgreSqlAgent, PostgreSqlQueryExecutor
+except ImportError:
+    skip = True
+else:
+    skip = False
+
+db_name = "postgresql_agent_test_db"
+dsn = "postgresql://postgres@localhost:5432/postgres"
+ddl = """\
+CREATE TABLE IF NOT EXISTS customer (
+    customer_id SERIAL NOT NULL,
+    first_name VARCHAR(45) NOT NULL,
+    last_name VARCHAR(45) NOT NULL,
+    email VARCHAR(50),
+    CONSTRAINT customer_pkey PRIMARY KEY (customer_id)
+    )
+"""
+def prepare_db():
+    with psycopg.connect(dsn, autocommit=True) as cnn:
+        with cnn.cursor() as cr:
+            cr.execute(f'DROP DATABASE IF EXISTS {db_name} WITH (FORCE)')
+            cr.execute(f'CREATE DATABASE {db_name}')
+            cr.execute(f"""
+                {ddl}
+                """)
+
+def clean_up_db():
+    with psycopg.connect(dsn, autocommit=True) as cnn:
+        with cnn.cursor() as cr:
+            cr.execute(f'DROP DATABASE IF EXISTS {db_name}')
+
+@pytest.mark.skipif(
+    skip or skip_openai,
+    reason="dependency is not installed OR requested to skip",
+)
+def test_postgresql_agent():
```

Comment:
@yonitjio can you please check, why this test is skipping?

Reply:
I believe it's because the `skip_openai` variable are set from `contrib-test.yml`.